libs/src/api/track.js

```diff
@@ -124,6 +124,77 @@ class Tracks extends Base {
     return trackId
   }
 
+  /**
+   * Takes in a readable stream if isServer is true, or a file reference if isServer is
```

Comment:
@hareeshnagaraj some more clarification about why this method should be used vs uploadTrack would be useful. we want to use this for album and playlist tracks, but the other method for 1...n individual tracks right?

Reply:
Good point, added some comments